src/js/components/covid19/budgetCategories/BudgetCategoriesCountTab.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * BudgetCategoriesCountTab.jsx
```

Comment:
Looks like we could reuse the `CountTab` component from agency v2. Eventually I might like to add tabs to the component library..

Reply:
Done!